core/src/main/resources/schema/app/20210922084527_CB-14183_alter_instance_metadata_table_add_variant_column.sql

```diff
@@ -0,0 +1,9 @@
+-- // CB-13685 alter image table add date column
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE instancemetadata ADD COLUMN IF NOT EXISTS variant VARCHAR(255);
```

Comment:
should we also update the current records?

Reply:
Nope, because we should know the cloudplatform for the default value. I think easier to handle if this value is null. If null is the same as the cloudpaltform name (or anything we want to want)